### ISSUE TITLE
Function expression order rule in order-in-model

### ISSUE THREAD

**Issue by cinkonaap** [netguru/eslint-plugin-netguru-ember#26]
Hey,

We are using the styleguide in our application, and we have a Model with custom method.

```
import DS from 'ember-data';

const { Model, attr, belongsTo } = DS;

export default Model.extend({
  a: attr('string'),
 
  b: belongsTo('c', { async: false }),

  convertA(paramA) {
    paramA === 'heyoo' ? this.get('a') : this.get('a') * 5;
  },
});
```

I wonder if having an custom method in model is an anti-pattern, or you omit `isFunctionExpression` in `order-in-model` for any other reason?

**Comment by netes** [comment]
In my opinion we should avoid defining custom methods in model definition files. This is a place for attributes and eventually computed properties, not for any other actions.

**Comment by cinkonaap** [comment]
@netes I agree to some degree, but what is strange that if there is an structure that has no priority in rule, it seems like it has to be on top. In example I've provided, `convertA` needs to be before `a`, which in my opinion should be changed. All not supported should structures should be at the bottom.

**Comment by michalsnik** [comment]
Hey, I agree @cinkonaap. All other cases should be totally valid when kept on the bottom, this is something we need to work on. Thanks for letting us know! :) I'll try to find some time in this week.

**Comment by michalsnik** [comment]
Fixed in `v1.6.4`
